plugins/inputs/sqlserver/connectionstring.go

```diff
@@ -0,0 +1,121 @@
+package sqlserver
+
+import (
+	"net"
+	"net/url"
+	"strings"
+)
+
+const emptyServerName = "<empty-server-name>"
```

Comment:
nit: Maybe it can go into a single const ( emptyServername = ""
                                                                 emptyDBName = ""
                                                                 otherVariables = "" 
                                                                 )

Reply:
Done.